### PR TITLE
chore: Move test-kustomize-build to .prow.yaml

### PR DESCRIPTION
**Pull request by tumido**
Part of: https://github.com/operate-first/toolbox/issues/18

With https://github.com/operate-first/toolbox/pull/13 we have the kustomize-build in toolbox now, switching to `presubmits` in `.prow.yaml`

/cc @goern @harshad16

### CODE REVIEW

**Review by harshad16 — COMMENTED** [review]
/lgtm

File: .prow.yaml [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+    decorate: true
+    max_concurrency: 1
+    run_if_changed: ".*yaml"
+    skip_report: false
+    context: aicoe-ci/prow/kustomize-build
+    spec:
+      containers:
+        - image: quay.io/operate-first/opf-toolbox:v0.3.1
+          command:
+            - "test-kustomize-build"
```
Comment (harshad16) [inline .prow.yaml:12]: is this to run from the image? if so, isn't the file kept at /usr/local/bin/ ?

**Review by harshad16 — CHANGES_REQUESTED** [review]
i just noticed this https://github.com/operate-first/toolbox/blob/cf04bd90a336eceb95e96b1f3362b45fb29c4954/Dockerfile#L49

File: .prow.yaml (hunk 1 above)
Comment (tumido) [reply .prow.yaml:12]: resolved in `v0.3.2` - can be called directly

**Review by harshad16 — COMMENTED** [review]
/lgtm

**Review by HumairAK — APPROVED** [review]

**Pull request merged**